### DISCUSSION TITLE
Finding beacons

### DISCUSSION THREAD

**Discussion by rlg2019** [Security-Onion-Solutions/securityonion#5079] [Unsupported Versions]
Good morning,

Is there a tutorial or blog post regarding finding beacons using Security Onion? Any links and/or tips would be appreciated. Thank you!

**Comment by weslambert** [comment]
You could use something like RITA, and ship your sensors logs to a dedicated server: 

https://github.com/activecm/rita

Alternatively, you could install on each sensor (or if running a standalone on the single node), however we recommend avoiding additional software on the Security Onion nodes wherever possible. 

https://docs.securityonion.net/en/2.3/rita.html

I haven't gotten a chance to continue working anymore with combining multiple datasets, but you could also experiment with some of what has been mentioned here for combining datasets: https://github.com/activecm/rita/issues/600. I'll try to write something up as I get time.
